DEPRECATIONS.md

```diff
@@ -0,0 +1,21 @@
+# Deprecations
+
+This document lists deprecated constructs in the SDK, with their expected time (⌛) of further deprecations and removals.
+
+| API / Feature | Deprecated (warning) | Deprecated (error) | Removed | Notes |
+| --- | --- | --- | --- | --- |
+| `setUser` (and similar) methods<br/>*client* | 2021.02.03.<br/>4.5.3 | 2021.05.03.⌛ | 2021.08.03.⌛ | Replaced by `connectUser` style methods that return `Call` objects, see migration guide (TODO) |
```

Comment:
We should update website docs, I believe `setUser` is still used there for anonymous and guest users

Reply:
Can you point me to where you still see `setUser` used? Those should be updated already, but if not, it's going on my TODO list.

Reply:
Can't find it anymore, somebody fixed it 👍